elasticache/memcached/Acornfile

```diff
@@ -31,13 +31,16 @@ services: admin: {
 }
 
 jobs: apply: {
+	memory: 512Mi
 	build: {
 		context:    "../"
 		dockerfile: "../memcached.Dockerfile"
 		additionalContexts: common: "../../libs"
 	}
 	files: "/app/config.json": std.toJSON(args)
 	env: {
+		GOGC:                         "25"
```

Comment:
Can these be put into the Docker images instead of the Acornfile?

Reply:
They could but it seems a bit weird to put some env vars there and the others here, no?

Reply:
In this case I don't think so because they are low level system configurations that feel like OS/Container environment settings. They aren't directly related to configuring and deploying the service in AWS. We also don't expect users to change these settings, as they are not runtime settings. So I feel pretty comfortable removing them from the Acornfile, it also doesn't help readability.

Reply:
Fair enough, moved them in the latest commit.